lib/grandpa/vote_message.go

```diff
@@ -225,7 +198,7 @@ func (s *Service) validateVoteMessage(from peer.ID, m *VoteMessage) (*Vote, erro
 
 	equivocated := s.checkForEquivocation(voter, just, m.Message.Stage)
 	if equivocated {
-		return nil, ErrEquivocation
+		return nil, fmt.Errorf("%w", ErrEquivocation)
```

Comment:
why are we wrapping the error here?

Reply:
[From https://engineering-handbook-pr-62.onrender.com/development/tech-stack/go#wrap-or-wrapped](https://engineering-handbook-pr-62.onrender.com/development/tech-stack/go#wrapping-the-sentinel-error)

> There are also cases where it is not necessary to add error context. For example if the database is 'closed', there is no need to add context really. In this case, use the format return fmt.Errorf("%w", ErrSentinel) and NOT return ErrSentinel. The reason is to prevent callers from using err == ErrDatabaseClosed, and force them to use errors.Is(err, ErrSentinel), so context can be added later on if needed.